packages/itwinui-css/src/global.scss

```diff
@@ -2,32 +2,19 @@
 // See LICENSE.md in the project root for license terms and full copyright notice.
 @use 'mixins';
 
-// reset anything that may have been set by v1
-:where(.iui-body, [class*='iui-theme-']) :where(.iui-root, [data-iui-theme]) {
-  :where([class*='iui-']:not(.iui-root, svg *)) {
-    &,
-    &::before,
-    &::after {
-      all: revert;
-    }
-  }
-}
-
 :where(.iui-root) {
   color: var(--iui-color-text);
   font-size: var(--iui-font-size-1);
   line-height: 1.5;
   font-family: var(--iui-font-sans);
 
-  // border-box on all iui- elements and their descendants under iui-root, excluding any nested iui-root elements
-  :where([class*='iui-']) {
-    &:where(:not(.iui-root)),
-    * {
-      &,
-      &::before,
-      &::after {
-        box-sizing: border-box;
-      }
+  // border-box on iui-root, all iui- elements and their descendants under it, excluding any nested iui-root elements
+  &,
```

Comment:
Looks like this change is breaking some visuals, presumably because we rely on `content-box` in some places.